<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::SettingsDialog</class>
 <widget class="QDialog" name="Qpid::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background:  #ebf4fa;
	border: 1px solid #697e8c;
}

#frameCaption {
	border-image: url(:/images/ui/top_caption.png);
}

#pushButtonClose {
	border-image: url(:/images/ui/close.png);
}

#pushButtonClose:hover {
	border-image: url(:/images/ui/close_hover.png);
}

#labelTitle {
	color: rgb(255,255,255);
	padding-left: 3px;
	font-size: 15px;
	font-weight: bold;
}

#labelPageHeader {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 #dfeef8, stop: 0.5 #d4e3ed, stop:1 #dfeef8);
	min-height: 30px;
	border-top: 1px solid gray;
	border-right: 1px solid gray;
	border-bottom: 1px solid #d2e1eb;
}

#widgetMain {
	border: 1px solid gray;
}

/* push button*/
#pushButtonOK,#pushButtonCancel {
	border: 0px;
	background-image: url(:/images/ui/button_bg.png);
	max-width: 130px;
	min-width: 130px;
	max-height: 30px;
	min-height: 30px; 
}

/* TreeWidget style sheet */
QTreeView {
	show-decoration-selected: 1;
	border: 1px solid gray;
}

QTreeView::item {
	min-height: 21px;
	padding: 5px;
	border-top-color: transparent;
	border-bottom-color: transparent;
}

QTreeView::item:selected {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 #dfeef8, stop: 0.5 #d4e3ed, stop:1 #dfeef8);
	border-top: 1px solid #d2e1eb;
	border-bottom: 1px solid #d2e1eb;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameCaption">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Voice Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="sysPanelContainer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="sysPanel" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="form" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widgetMain" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="indexTree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>0</number>
           </attribute>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetPageContainer" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelPageHeader">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="pages">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="Qpid::SettingsGeneralPage" name="generalPage"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetButtonContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>470</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonOK">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qpid::SettingsGeneralPage</class>
   <extends>QWidget</extends>
   <header>settingsgeneralpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>indexTree</tabstop>
 </tabstops>
 <resources>
  <include location="../phone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
